lms/templates/instructor/instructor_dashboard_2/data_download_2/problem_report.html

```diff
@@ -0,0 +1,37 @@
+<%page args="section_data" expression_filter="h"/>
+<%namespace name='static' file='/static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<section id="problem" class="idash-section tab-data" aria-labelledby="header-problem">
+    <h6 class="mb-20" id="header-problem">
+        ${_("Select a problem to generate a CSV \
+        file that lists all student answers to the problem. You also select a section or chapter to include \
+        results of all problems in that section or chapter.")}
+    </h6>
+    <div class="mb-15 problems">
+        ${static.renderReact(
+        component="ProblemBrowser",
+        id="react-block-listing",
+        props={
+        "courseId": course.id,
+        "excludeBlockTypes": ['html', 'video', 'discussion']
+        }
+        )}
```

Comment:
Why react here?

Reply:
The problem selector was already built component which is reused here.